app/src/main/java/org/mozilla/fenix/onboarding/WallpaperOnboardingDialogFragment.kt

```diff
@@ -128,5 +129,11 @@ class WallpaperOnboardingDialogFragment : BottomSheetDialogFragment() {
 
     companion object {
         const val THUMBNAILS_SELECTION_COUNT = 6
```

Comment:
```suggestion
        // The number of wallpaper thumbnails to display.
        const val THUMBNAILS_SELECTION_COUNT = 6
```

Let's add a comment for this while we are at it.